### PR TITLE
Add Microsoft.NetworkAnalytics RP SDK package

### PR DESCRIPTION
**Pull request by gaagarwalMS**
## Adding SDK package and the generated code for the Microsoft.NetworkAnalytics RP

Test Results:
image

### CODE REVIEW

**Comment by jsquire** [comment]
@gaagarwalMS:  Our automation is recognizing you as a community member.  Please review the Azure SDK onboarding documentation and ensure that your GitHub organization membership and permissions are correct before moving this out of draft. 

- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

File: sdk/hdinsight/ci.mgmt.yml [hunk 1]
```diff
@@ -11,7 +11,6 @@ pr:
   paths:
     include:
     - sdk/hdinsight/ci.mgmt.yml
-    - sdk/hdinsight/Azure.ResourceManager.HDInsight/
```
Comment (ArthurMa1978) [inline sdk/hdinsight/ci.mgmt.yml:14]: revert this change

**Review by ArthurMa1978 — DISMISSED** [review]
This is the first release for your service, did you get the namespace review for it? If yes can you provide the like of the [request](https://github.com/Azure/azure-sdk-pr/issues/new?assignees=kyle-patterson%2C+ronniegeraghty&labels=architecture%2C+board-review%2C+mgmt-namespace-review&projects=&template=adp_mgmt_namespace_review.md&title=Board+Review%3A+Management+Plane+Namespace+Review+%3Cclient+library+name%3E)? If no, please file the request for review.

File: sdk/networkanalytics/Azure.ResourceManager.NetworkAnalytics/src/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: Azure.Core.AzureResourceProviderNamespace("NetworkAnalytics")]
```
Comment (ArthurMa1978) [inline sdk/networkanalytics/Azure.ResourceManager.NetworkAnalytics/src/Properties/AssemblyInfo.cs:6]: Microsoft.NetworkAnalytics

File: sdk/networkanalytics/Azure.ResourceManager.NetworkAnalytics/README.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+# Microsoft Azure NetworkAnalytics management client library for .NET
+
+**[Describe the service briefly first.]**
```
Comment (ArthurMa1978) [inline sdk/networkanalytics/Azure.ResourceManager.NetworkAnalytics/README.md:3]: Describe your service here

**Comment by gaagarwalMS** [comment]
> This is the first release for your service, did you get the namespace review for it? If yes can you provide the like of the [request](https://github.com/Azure/azure-sdk-pr/issues/new?assignees=kyle-patterson%2C+ronniegeraghty&labels=architecture%2C+board-review%2C+mgmt-namespace-review&projects=&template=adp_mgmt_namespace_review.md&title=Board+Review%3A+Management+Plane+Namespace+Review+%3Cclient+library+name%3E)? If no, please file the request for review.

Yes, we have got the namespace approval for our service. Please find the link here:
[Mgmt Namespace Approval for Azure Operator Insights · Issue #867 · Azure/azure-sdk-pr (github.com)](https://github.com/Azure/azure-sdk-pr/issues/867)

File: sdk/networkanalytics/Azure.ResourceManager.NetworkAnalytics/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```
Comment (ArthurMa1978) [inline sdk/networkanalytics/Azure.ResourceManager.NetworkAnalytics/CHANGELOG.md:3]: Please specify the release date you expected by replacing the `Unreleased` , format is `yyyy-mm-dd`.

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**